resources/js/Pages/Authors.vue

```diff
@@ -0,0 +1,160 @@
+<script setup>
+import Papa from 'papaparse'
+import moment from 'moment'
+import { computed, ref, watch } from 'vue'
+import { useLogsStore } from '@/Stores/logsStore'
+import { Head } from '@inertiajs/vue3'
+import resolveConfig from 'tailwindcss/resolveConfig'
+import tailwindConfig from '../../../tailwind.config.js'
```

Comment:
If your tailwind Config file is configured correctly you don't need to import it on every file you want to use it. It's global file working everywhere.

Reply:
I'm talking about this : `import tailwindConfig from '../../../tailwind.config.js`

sorry for marking all of 4 lines

Reply:
How can I access the properties without importing the config?